sdk/web-pubsub/arm-webpubsub/review/arm-webpubsub.api.md

```diff
@@ -579,15 +597,16 @@ export interface WebPubSub {
     beginDeleteAndWait(resourceGroupName: string, resourceName: string, options?: WebPubSubDeleteOptionalParams): Promise<void>;
     beginRegenerateKey(resourceGroupName: string, resourceName: string, parameters: RegenerateKeyParameters, options?: WebPubSubRegenerateKeyOptionalParams): Promise<SimplePollerLike<OperationState<WebPubSubRegenerateKeyResponse>, WebPubSubRegenerateKeyResponse>>;
     beginRegenerateKeyAndWait(resourceGroupName: string, resourceName: string, parameters: RegenerateKeyParameters, options?: WebPubSubRegenerateKeyOptionalParams): Promise<WebPubSubRegenerateKeyResponse>;
-    beginRestart(resourceGroupName: string, resourceName: string, options?: WebPubSubRestartOptionalParams): Promise<SimplePollerLike<OperationState<void>, void>>;
-    beginRestartAndWait(resourceGroupName: string, resourceName: string, options?: WebPubSubRestartOptionalParams): Promise<void>;
+    beginRestart(resourceGroupName: string, resourceName: string, options?: WebPubSubRestartOptionalParams): Promise<SimplePollerLike<OperationState<WebPubSubRestartResponse>, WebPubSubRestartResponse>>;
+    beginRestartAndWait(resourceGroupName: string, resourceName: string, options?: WebPubSubRestartOptionalParams): Promise<WebPubSubRestartResponse>;
```

Comment:
Could you share the change for swagger?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/22948/commits/8de978c86b3142a3ac5bc7de8cd2c4c74d7faeaf#diff-f1b30594d5c33a24f727bebbe81bd04704cc186b09f8a6b0b575faba88d412adR1841